src/parsers/eventParser.js

```diff
@@ -66,73 +88,149 @@ export const isSqsEvent = event => {
   for (const key of Object.keys(event)) {
     if (!API_GW_KEYS_ORDER.includes(key) && !API_GW_KEYS_DELETE_KEYS.includes(key)) {
-      parsed_event[key] = event[key];
+      parsedEvent[key] = event[key];
     }
   }
-  return parsed_event;
+  return parsedEvent;
 };
 
 export const parseSnsEvent = event => {
-  const new_sns_event = {};
-  new_sns_event['Records'] = [];
+  const newSnsEvent = {};
+  newSnsEvent['Records'] = [];
   // Add order keys
   for (const rec of event['Records']) {
-    const new_sns_record_event = {};
+    const newSnsRecordEvent = {};
     for (const key of SNS_KEYS_ORDER) {
       if (rec.Sns != null && rec.Sns[key] != null) {
-        new_sns_record_event[key] = rec.Sns[key];
+        newSnsRecordEvent[key] = rec.Sns[key];
       }
     }
-    new_sns_event['Records'].push({ Sns: new_sns_record_event });
+    newSnsEvent['Records'].push({ Sns: newSnsRecordEvent });
   }
-  return new_sns_event;
+  return newSnsEvent;
 };
 
 export const parseSqsEvent = event => {
-  const new_sqs_event = {};
-  new_sqs_event['Records'] = [];
+  const newSqsEvent = {};
+  newSqsEvent['Records'] = [];
   // Add order keys
   for (const rec of event['Records']) {
-    const new_sqs_record_event = {};
+    const newSqsRecordEvent = {};
     for (const key of SQS_KEYS_ORDER) {
       if (rec[key] != null) {
-        new_sqs_record_event[key] = rec[key];
+        newSqsRecordEvent[key] = rec[key];
       }
     }
-    new_sqs_event['Records'].push(new_sqs_record_event);
+    newSqsEvent['Records'].push(newSqsRecordEvent);
   }
-  return new_sqs_event;
+  return newSqsEvent;
+};
+
+export const parseS3Event = event => {
+  const newS3Event = {};
+  newS3Event['Records'] = [];
+  // Add order keys
+  for (const rec of event['Records']) {
+    const newS3RecordEvent = {};
+    for (const key of S3_KEYS_ORDER) {
+      if (rec.hasOwnProperty(key) != null) {
+        newS3RecordEvent[key] = rec[key];
+      }
+    }
+    if (rec.hasOwnProperty('s3')) {
+      newS3RecordEvent.s3 = {};
+      if (rec.s3.hasOwnProperty('bucket')) {
+        newS3RecordEvent.s3.bucket = {};
+        for (const key of S3_BUCKET_KEYS_ORDER) {
+          newS3RecordEvent.s3.bucket[key] = rec.s3.bucket[key];
+        }
+      }
+      if (rec.s3.hasOwnProperty('object')) {
+        newS3RecordEvent.s3.object = {};
+        for (const key of S3_OBJECT_KEYS_ORDER) {
+          newS3RecordEvent.s3.object[key] = rec.s3.object[key];
+        }
+      }
+    }
+    newS3Event['Records'].push(newS3RecordEvent);
+  }
+  return newS3Event;
+};
+
+export const parseCloudfrontEvent = event => {
+  const newCloudfrontEvent = {};
+  newCloudfrontEvent['Records'] = [];
+  // Add order keys
+  for (const rec of event['Records']) {
+    const cfRecord = rec['cf'] || {};
+    const newCloudfrontRecordEvent = {};
+    for (const key of CLOUDFRONT_KEYS_ORDER) {
+      if (cfRecord.hasOwnProperty(key) != null) {
+        newCloudfrontRecordEvent[key] = cfRecord[key];
+      }
+    }
```

Comment:
You can do util that will help you reuse this?
That gets a JSON and order and return a JSON